package.json

```diff
@@ -8,7 +8,8 @@
     "lint": "eslint src test",
     "test": "npm run lint && mocha test/**/*-test.js",
     "watch": "npm test -- -w",
-    "lint-fix": "eslint src test --fix"
+    "lint-fix": "eslint src test --fix",
+    "windows-debug-test": "@powershell -Command $env:DEBUG='http:*'; npm test"
```

Comment:
Well, the best practice is to avoid using platform-dependent commands in `package.json`. It's better to use the `cross-env` package for this purpose.